tools/cli/adminDBScanCommand.go

```diff
@@ -141,11 +147,12 @@ type (
 	// ProgressReport contains metadata about the scan for all shards which have been finished
 	// This is periodically printed to stdout
 	ProgressReport struct {
-		NumberOfShardsFinished           int
+		NumberOfShardsFinished           int64
 		TotalExecutionsCount             int64
 		CorruptedExecutionsCount         int64
 		ExecutionCheckFailureCount       int64
 		NumberOfShardScanFailures        int64
+		ShardsFailed                     []int
```

Comment:
If # of shards has type `int64`, then this one should have type `[]int64`. Need to be consistent.